src/expressApp.js

```diff
@@ -108,6 +114,16 @@ function configureProxy(app, { env }) {
     proxyReq.setHeader('User-Agent', userAgent);
   });
 
+  proxy.on('error', (err, _, res) => {
+    debug('Proxy server error: ', err);
+    res.status(500).send('BitGo Express encountered an error while attempting to proxy your request to BitGo. Please try again.');
```

Comment:
we should return sth json-parsable that is more similar to our regular 500 errors